src/main/java/net/minecraftforge/common/config/ConfigElement.java

```diff
@@ -360,4 +360,40 @@ public Object getMaxValue()
     {
         return isProperty ? prop.getMaxValue() : null;
     }
+    
+    public static ConfigElement from(String modid)
+    {
+        return from(modid, null);
```

Comment:
why would this ever be called with a null name?  They should ateleast know what config they are trying to read, or pull in a Class and deal with it that way.